### PR TITLE
Update default_embedding_functions.md

### PR DESCRIPTION
**Pull request by asmith26**
Fix example (tried to make more similar to other examples too).

### CODE REVIEW

File: docs/src/embeddings/default_embedding_functions.md [hunk 1]
```diff
@@ -186,21 +186,26 @@ We offer support for all huggingface models (which can be loaded via [transforme
...
-
 from lancedb.embeddings import get_registry
 from lancedb.pydantic import LanceModel, Vector
 
+db = lancedb.connect("/tmp/db")
 model = get_registry().get("huggingface").create(name='facebook/bart-base')
 
-class TextModel(LanceModel):
+
+class Words(LanceModel):
```
Comment (raghavdixit99) [inline docs/src/embeddings/default_embedding_functions.md:196]: Can we keep TextModel ?
the name is more apt than 'Words'

File: docs/src/embeddings/default_embedding_functions.md [hunk 2]
```diff
@@ -186,21 +186,26 @@ We offer support for all huggingface models (which can be loaded via [transforme
...
-query = "old greeting"
+
+table = db.create_table("words", schema=Words)
+table.add(
+    [
+        {"text": "hello world"},
+        {"text": "goodbye world"}
+    ]
+)
+query = "greetings"
```
Comment (raghavdixit99) [inline docs/src/embeddings/default_embedding_functions.md:208]: both are greetings, we could show a more clear example. Either the older example or something simple yet distinct , for ex :

```python
table.add(
    [
        {"text": "Golden Retriever energy"},
        {"text": "Black cat"}
    ]
)
query = "playful dog"
```

Comment (asmith26) [reply docs/src/embeddings/default_embedding_functions.md:208]: Thanks for the comment, happy to use this - would you like me to change all the other examples to match this, i.e.:

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L169-L176

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L202-L203

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L231-L238

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L287-L291

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L322-L327

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L372-L377

File: docs/src/embeddings/default_embedding_functions.md [hunk 3]
```diff
@@ -186,21 +186,26 @@ We offer support for all huggingface models (which can be loaded via [transforme
 Example usage - 
 ```python
 import lancedb
-import pandas as pd
-
 from lancedb.embeddings import get_registry
 from lancedb.pydantic import LanceModel, Vector
 
+db = lancedb.connect("/tmp/db")
```
[the remark this answers is not in the document]
Comment (asmith26) [reply docs/src/embeddings/default_embedding_functions.md:192]: Thanks for the coment, did you mean `./lancedb`? Happy to use this - would you like me to change all the other examples to match this, i.e.:

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L161

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L223

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L284

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L323

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L373

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L404


https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L484

File: docs/src/embeddings/default_embedding_functions.md (hunk 1 above)
Comment (asmith26) [reply docs/src/embeddings/default_embedding_functions.md:196]: Thanks for the comment, happy to use this - would you like me to change all the other examples to match this, i.e.:

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L164

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L226

https://github.com/lancedb/lancedb/blob/fdb5d6fdf1454a1c545cc9da53abad8d1ccb6b99/docs/src/embeddings/default_embedding_functions.md?plain=1#L280

File: docs/src/embeddings/default_embedding_functions.md (hunk 2 above)
Comment (raghavdixit99) [reply docs/src/embeddings/default_embedding_functions.md:208]: yes sure wherever there is less clarity you can change there, thanks a lot !

File: docs/src/embeddings/default_embedding_functions.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (raghavdixit99) [reply docs/src/embeddings/default_embedding_functions.md:192]: Yes, that would be great to maintain consistency, thank you !

File: docs/src/embeddings/default_embedding_functions.md (hunk 1 above)
Comment (raghavdixit99) [reply docs/src/embeddings/default_embedding_functions.md:196]: Please change only where `Words` is the class name, Schema is ok, Thank you

**Comment by asmith26** [comment]
Looks like there have been some great updates to this, so I'll close this, thanks!

**Pull request closed without merging**